src/datasources/cache/cache.first.data.source.ts

```diff
@@ -24,7 +24,7 @@ import {
  */
 @Injectable()
 export class CacheFirstDataSource {
-  private readonly ERROR_TTL_SECONDS = 30;
+  private readonly defaultNotFoundErrorTTLSeconds = 30;
```

Comment:
I would not keep this one. Else we are splitting the configuration across multiple stages:
- The config from the envar
- The default if the envar is not set
- The default if it does not apply to token or contracts.

I'd move the default to the config also.

Reply:
Yeah, I agree, initially I tried to not touch this wider default and just extract the contracts and tokens ones, targetting a further refactor in the future. But yes, maybe it's better to tackle this duplication/confusion now.

Reply:
Changed in [4e30dc6](https://github.com/safe-global/safe-client-gateway-nest/pull/578/commits/4e30dc61d72b12d7b8baa789be526cca80bca083)